aten/src/ATen/record_function.cpp

```diff
@@ -88,8 +89,64 @@ void set_record_function_tls_(const RecordFunctionTLS& tls) {
   rf_tls() = tls;
 }
 
+enum class ToggledCallbackResult {
+  NotFound,
+  FoundButNotToggled,
+  FoundAndToggled,
+};
+
+template <typename RecordFunctionCallbacks>
+static ToggledCallbackResult findAndToggleCallback(
+    RecordFunctionCallbacks& cbs, CallbackHandle handle, bool enabled) {
+  auto it = std::find_if(
```

Comment:
yeah, we could use the fact that the vector is sorted, but this indeed is not supposed to be called in the hot path, also we don't have that many callbacks anyways